doc/flash-manipulation.md

```diff
@@ -0,0 +1,136 @@
+# SMC flash manipulation tools, v6
+Author: Eirik Stolpe
+
+These tools allows X16 to dump entire SMC flash, and manipulate bootloader.
```

Comment:
`By reading this document, it is not straight forward to know what is referred to as "these tools".

Could it be an idea to move this document to the "tools" folder?

By the way. The tools in the "tools" folder are incomplete. The "SMCBLW16.PRG" tool (which updates the bootloader) expects a healthy bootloader v2 to be present in the same folder, as "BOOT2.BIN".

Reply:
Oh, you have changed the instructions to refer to bootloader v3 instead. If you think v3 is ready to be rolled out in the initial version of this document, we should also add its CRC16 to the bootloader validator and programming tools (and bump their versions).